src/ILCompiler.CppCodeGen/src/CppCodeGen/ILToCppImporter.cs

```diff
@@ -1163,6 +1163,43 @@ private bool ImportIntrinsicCall(MethodDesc method)
                         return true;
                     }
                     break;
+                case "DefaultConstructorOf":
+                    if (IsTypeName(method, "System", "Activator") && method.Instantiation.Length == 1)
+                    {
+                        string expr;
+
+                        if (runtimeDeterminedMethod.IsRuntimeDeterminedExactMethod)
+                        {
+                            expr = string.Concat(
+                                "(intptr_t)",
+                                GetGenericLookupHelperAndAddReference(ReadyToRunHelperId.DefaultConstructor, runtimeDeterminedMethod.Instantiation[0]),
+                                "(", GetGenericContext(), ")");
+                        }
+                        else
+                        {
+                            MethodDesc ctor = method.Instantiation[0].GetDefaultConstructor();
```

Comment:
Can this use `Compilation.ComputeConstantLookup` so that we don't duplicate the `ClassWithMissingConstructor` logic? You'll need to cast the returned `ISymbolNode` to `IMethodNode` to get to the method, but it should work.

Reply:
Thank you! Done.